upgrading/rosa-hcp-upgrading.adoc

```diff
@@ -6,20 +6,13 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-[id="rosa-hcp-lifecycle-policy_{context}"]
-== Life cycle policies and planning
+You can upgrade {hcp-title-first} clusters by using individual upgrades through the {product-title} (ROSA) command line interface (CLI), `rosa`. You can also specify if you want to upgrade just the control plane, a single machine pool, or all machine pools and the control plane.
 
-To plan an upgrade, review the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle]. The life cycle page includes release definitions, support and upgrade requirements, installation policy information and life cycle dates.
-
-You can manually upgrade your cluster. Red Hat Site Reliability Engineers (SREs) monitor upgrade progress and remedy any issues encountered.
-
-[id="rosa-hcp-upgrading-a-cluster"]
-== Upgrading a ROSA cluster
-You can upgrade {hcp-title-first} clusters by using individual upgrades through the {product-title} (ROSA) CLI, `rosa`.
+To plan an upgrade, review the xref:../rosa_architecture/rosa_policy_service_definition/rosa-hcp-life-cycle.adoc#rosa-hcp-life-cycle[{hcp-title} update life cycle]. The life cycle page includes release definitions, support and upgrade requirements, installation policy information, and life cycle dates.
 
 [NOTE]
 ====
-When following a scheduled upgrade policy, there might be a delay of no more than thirty minutes before the upgrade process begins, even if it is an immediate upgrade. Additionally, the duration of the upgrade might vary based on your workload configuration and with machine pool upgrades, the number of worker nodes.
+The duration of the upgrade might vary based on your workload configuration and with machine pool upgrades, the number of worker nodes.
```

Comment:
```suggestion
Cluster upgrade duration varies based on your workload configuration and machine pool upgrade duration varies based on the number of worker nodes.
```
Not sure if it should be "cluster" or "hosted control plane" at the start of the sentence.